<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xmodeltest</class>
 <widget class="QMainWindow" name="xmodeltest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#eee;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="tool_open_msa">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #aaa;
background-color: #272;</string>
            </property>
            <property name="text">
             <string>msa</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_open_tree">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>tree</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_open_parts">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>partitions</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="tool_run_settings">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_results">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>results</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_help">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>help!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_reset">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../mtgraphics.qrc">:/IMG/IMG/mt-icon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTextBrowser" name="consoleRun">
          <property name="font">
           <font>
            <family>Courier 10 Pitch</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
         <widget class="QTextBrowser" name="consoleMeta">
          <property name="font">
           <font>
            <family>Courier 10 Pitch</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#555;color:#aaa;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background:#555;color:#aaa;</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mnu_open_msa"/>
    <addaction name="mnu_open_tree"/>
    <addaction name="mnu_open_parts"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #666;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open_msa"/>
   <addaction name="act_open_tree"/>
   <addaction name="act_open_parts"/>
   <addaction name="separator"/>
  </widget>
  <action name="act_open_msa">
   <property name="icon">
    <iconset resource="../mtgraphics.qrc">
     <normaloff>:/IMG/IMG/icon-openMSA.png</normaloff>:/IMG/IMG/icon-openMSA.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="act_open_tree">
   <property name="icon">
    <iconset resource="../mtgraphics.qrc">
     <normaloff>:/IMG/IMG/icon-opentree.png</normaloff>:/IMG/IMG/icon-opentree.png</iconset>
   </property>
   <property name="text">
    <string>Load tree</string>
   </property>
  </action>
  <action name="act_open_parts">
   <property name="icon">
    <iconset resource="../mtgraphics.qrc">
     <normaloff>:/IMG/IMG/icon-openparts.png</normaloff>:/IMG/IMG/icon-openparts.png</iconset>
   </property>
   <property name="text">
    <string>Load Partitions</string>
   </property>
  </action>
  <action name="mnu_open_msa">
   <property name="text">
    <string>Load MSA</string>
   </property>
  </action>
  <action name="mnu_open_tree">
   <property name="text">
    <string>Load Tree</string>
   </property>
  </action>
  <action name="mnu_open_parts">
   <property name="text">
    <string>Load Partitions</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../mtgraphics.qrc"/>
 </resources>
 <connections/>
</ui>
